src/main/java/sirius/biz/storage/layer1/ObjectStorage.java

```diff
@@ -52,6 +53,10 @@
      * to use for a space.
      */
     public static final String CONFIG_KEY_LAYER1_CIPHER = "cipher";
+    /**
+     * Contains the name of the storage space used for "work files".
+     */
+    public static final String WORK_SPACE_NAME = "work";
```

Comment:
I'd say this either belongs to layer2 or 3 or we move this into some class in utils?